lib/glific/flows/router.ex

```diff
@@ -156,7 +156,7 @@ defmodule Glific.Flows.Router do
       router.cases
       |> Enum.reduce(
         errors,
-        &Case.validate(&1, &2, flow)
+        &Case.validate(&1, &2, flow, is_nil(router.wait))
```

Comment:
Semantically this is not correct. Either send router.wait as a true value or fix the Case.validate cluase.

Reply:
@pankaj-ag made requested changes